docs/users_guide/landice/framework.rst

```diff
@@ -0,0 +1,105 @@
+.. _landice_framework:
+
+Land-ice Framework
+==================
```

Comment:
I don't think much of this page belongs in the User's Guide.  Conceptually, a user is someone who installed the `compass` conda package and is running the test cases just using a user config file (if needed) and a local clone of E3SM or MALI-Dev.  They can change config options either before or after setting up a test case.  They can also edit namelist and streams files (if they're a "power-user" .  But that's basically it.  They don't necessarily have access to or interest in the code, so this is basically an area for describing how to set things up and run them.

So in terms of framework, the main focus here should be on the config options that apply across test cases that use the framework.  This is done on the ocean side mostly to avoid redundantly describing the same types of config options for each test group that uses them.  I'm not sure how much that applies in your case.

Reply:
addressed in [d2ba3d7](https://github.com/MPAS-Dev/compass/pull/590/commits/d2ba3d738efd596b9f5538a09f1114283d4411f2)